src/FSharp.Data.GraphQL.Shared/TypeSystem.fs

```diff
@@ -1536,6 +1543,17 @@ and DirectiveDef =
       /// Array of arguments defined within that directive.
       Args : InputFieldDef [] }
 
+/// Field metadata definition.
+and Metadata() =
```

Comment:
This is the metadata implementation, just a façade for a Dictionary<string, obj>.